docs/docs/domain.mdx

```diff
@@ -125,26 +158,36 @@ Make this a nice table instead.
         type: text
   ```
 
-
-
 * **Description**
 
-  Results in the feature of the slot being set to `1` if any value is set.
-  Otherwise the feature will be set to `0` (no value is set).
+  If `influence_conversation` is set to `true`, the assistant's behavior will change
+  depending on whether the slot is set or not. Different texts do not influence the
```

Comment:
```suggestion
  depending on whether the slot is set or not. Different slot values do not influence the
```

Reply:
I chose `texts` as different slot values _do_ influence the prediction (if the value is `None`).

Reply:
👍